main.go

```diff
@@ -55,10 +56,27 @@ var infoTable *tview.Table
 var debugText *tview.TextView
 var tree *tview.TreeView
 
+var (
+	_ = determineWorkingDirectory()
+	configFile = "config.json"
+)
+
+func determineWorkingDirectory() string {
```

Comment:
why does `determineWorkingDirectory()` return string?

Reply:
I would also suggest changing the function name to something like `moveToWorkingDirectory()`